webnode/src/components/storage/index.tsx

```diff
@@ -16,45 +20,48 @@ import ConsentOverlay from "../consent-overlay";
 
 import LOGO from "../../assets/images/logo.svg";
 
-const GenesisHashInput = onChange => (
-  <Input
-    style={{ width: 1000, paddingBottom: 20 }}
-    onChange={onChange}
-    label={{ tag: true, content: "Genesis Hash" }}
-    labelPosition="right"
-    placeholder=""
-  />
-);
+interface StorageProps {
+  treasures: any;
+  numberOfCalls: number;
+  statuses: any;
+  status: string;
+  consent: string;
+  giveConsent: () => any;
+  denyConsent: () => any;
+  findTreasure: (obj: FindTreasureType) => any;
```

Comment:
How do these custom types work?

Reply:
in TypeScript, you need to define the uploaded object and its types if you do not first see all the attributes of the function and send the entire wrapped object